.travis.yml

```diff
@@ -5,3 +5,5 @@ node_js:
   - "0.12"
   - "0.10"
 sudo: false
+after_success:
+  - bash <(curl -s https://codecov.io/bash)
```

Comment:
I know this is already merged, but I'm not sure we want to run a script we don't control like this.  What do you folks think of rolling this back and auditing the script?  If its safe and does things we're comfortable with, then we could just put it in our repo.